src/main/webapp/event-form.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/css/main.css">
+  </head>
+  <body>
+    <h2>Create your event!</h2>
+    <form action="" method="POST">
```

Comment:
I'm a bit rusty on how HTML interacts with the Java files, but should there be an action specified? (I was thinking `/event-form`, but I'm not entirely sure.) Thoughts?